### DISCUSSION TITLE
Calculte Differential accessible regions for single-cell ATAC and Bulk ATAC

### DISCUSSION THREAD

**Discussion by 1221li** [GreenleafLab/ArchR#1376] [Feature Requests]
Dear author, I can successfully use single-cell ATAC of different samples to analyze the Differential accessible regions, but NOW I want to calculate the Differential accessible regions between single-cell ATAC of sample 1 and Bulk ATAC of sample 2 by ARCHR, I have no clue about this, I would like to ask, can ARCHR answer this question？
I am looking forward to your reply ！
best wishes！

**Comment by rcorces** [comment]
No. ArchR cannot handle bulk ATAC-seq. You will need to analyze scATAC with ArchR, bulk ATAC with something else, and use custom scripts to compare them.

**Reply by 1221li** [reply]
Thank you very much for your reply. If I have single cell ATAC and bulk ATAC data of the same sample, can I use ARCHR to get the  Differential accessible regions ?
